src/collectives.c

```diff
@@ -109,16 +108,14 @@ shmem_internal_circular_iter_next(int curr, int PE_start, int logPE_stride, int
 
 
 int
-shmem_internal_collectives_init(int requested_crossover,
-                                int requested_radix)
+shmem_internal_collectives_init(void)
 {
     int i, j, k;
     int tmp_radix;
     int my_root = 0;
     char *type;
 
-    tree_radix = requested_radix;
-    shmem_internal_tree_crossover = requested_crossover;
+    tree_radix = (int) shmem_internal_params.COLL_RADIX;
```

Comment:
Could we just change ```tree_radix``` to a long and avoid the cast?  Or would that change propagate too deeply?